pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -406,6 +406,24 @@ type MonitoringSpec struct {
 	// Interval determines prometheus scrape interval
 	// +optional
 	Interval *metav1.Duration `json:"interval,omitempty"`
+
+	// Ceph exporter configuration
+	// +optional
+	Exporter *CephExporterSpec `json:"exporter,omitempty"`
+}
+
+type CephExporterSpec struct {
+	// Only performance counters greater than or equal to this option are fetched
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=10
+	// +kubebuilder:default=5
+	PerfCountersPrioLimit uint `json:"perfCountersPrioLimit,omitempty"`
```

Comment:
Ceph documentation does not mention `--prio-limit` or `exporter_prio_limit`. Given that `exporter_stats_period` is the way Ceph documents the other value, I think it would be helpful for users here to document that this value will be applied to `exporter_stats_period`.

I think adding `PerfCounters` as a prefix adds some helpful clarity around what `PrioLimit` means, so I'm happy to keep that for the help it adds.